src/modules/components/vote/voteDataListItem/voteDataListItemStructure/voteDataListItemStructure.tsx

```diff
@@ -66,11 +73,20 @@ export const VoteDataListItemStructure: React.FC<IVoteDataListItemStructureProps
                     )}
                     {isCurrentUser && <Tag variant="neutral" label={copy.voteDataListItemStructure.you} />}
                 </span>
-                {isTokenVoting && <span className="truncate text-neutral-500">{formattedTokenVote}</span>}
+                {isTokenVoting && (
+                    <span className="truncate text-sm text-neutral-500 md:text-base">{formattedTokenVote}</span>
+                )}
+            </div>
+
+            <div className="flex items-center gap-x-1 text-sm font-normal leading-tight text-neutral-500 md:gap-x-1.5 md:text-base">
+                <span>{confirmationLabel ?? copy.voteProposalDataListItemStructure.voted}</span>
```

Comment:
Each translation is scoped for each component
```suggestion
                <span>{confirmationLabel ?? copy.voteDataListItemStructure.voted}</span>
```